keps/sig-windows/2258-node-log-query/README.md

```diff
@@ -231,29 +231,30 @@ On Windows node, the same kubelet service logs will queried by explicitly
 specifying the log file. In Linux the explicit log file query will be tested by
 querying a random file in present in /var/log.
 
-On the Linux side tests will be added to [kubelet node](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/node/kubelet.go)
-e2e tests. For Windows a new set of tests will be added to the existing
-[e2e tests](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/windows).
+Tests have been added to [kubelet node](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/node/kubelet.go)
+e2e tests that are run as a daily periodic job:
+- https://testgrid.k8s.io/sig-windows-master-release#capz-master-windows-alpha-nodelogquery
```

Comment:
this is windows? where's the linux job?

Reply:
This jobs runs tests against both Linux and Windows nodes.

Reply:
I have explicitly called this out in the KEP now. PTAL.

Reply:
This job cannot be release blocking quality for GA, FYI.

Because Azure jobs are not on a community owned account (see release blocking jobs policy: https://github.com/kubernetes/sig-release/blob/master/release-blocking-jobs.md) ... and these jobs are actually at risk of being shut down long term as we shut down the google cluster that has the unknown third-party secret for the account. (Which we're not doing in community clusters, ongoing discussions in SIG Testing / K8s Infra)

Reply:
Noted @BenTheElder. BTW, this feature will never be enabled by default. So does it need to be part of the release blocking job? Or are you saying this has to be moved off the Azure jobs irrespective? This would affect other sig-windows e2e tests too.

cc: @marosset @jsturtevant

Reply:
> and these jobs are actually at risk of being shut down long term as we shut down the google cluster that has the unknown third-party secret for the account as part of finishing migrating to CI fully on K8s Infra clusters. (In community clusters we're not letting third parties add arbitrary secrets unlike the old Google clusters, ongoing discussions in SIG Testing / K8s Infra)

Where can we find more information about this? This will affect the CAPZ and image-builder jobs as well. cc: @jackfrancis

Reply:
> Where can we find more information about this? This will affect the CAPZ and image-builder jobs as well. cc: @jackfrancis

We've (SIG K8s Infra + Testing) been talking to various folks (eg lachlan, jeremy, bridgett) at multiple venues for O(years) now but concretely hammering something out is ongoing, previously we'd hoped to get an Azure account properly under the project instead of going that route, but for recent discussion see this doc from SIG Testing + K8s Infra meetings this year: https://docs.google.com/document/d/1erBhuCwY26d0UfPbzt8lEj6bYT2hOUKzc2j36YHVqfM/edit

Thanks for CC-ing more folks, but please let's discuss _that_ aspect in #sig-k8s-infra or similar, it's an off-topic FYI as far as this KEP is concerned.

Aside from that, more on-topic FYI: This job cannot be release blocking and has not been for a long time now so if so if you want to GA this feature later you're going to need a different job for release blocking coverage.

Reply:
Going to keep this thread on topic 😄 

>This job cannot be release blocking and has not been for a long time now so if so if you want to GA this feature later you're going to need a different job for release blocking coverage.

Understood. I will figure out next steps before we get to GA starting with the next release.